src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -18,8 +18,8 @@
 	<ItemGroup>
```

Comment:
I'm not sure `feat:` is the correct conventional commit type. It seems `fix:` would be more appropriate.